fn_phish_tank/README.md

```diff
@@ -0,0 +1,132 @@
+# PhishTank Function for IBM Resilient  
+  
+## Table of Contents    
+- [About This Package](#about-this-package)  
+ - [Prerequisites](#prerequisites)  
+ - [Installation](#installation)  
+ - [Function Inputs](#function-inputs)  
+ - [Function Output](#function-output)  
+ - [Pre-Process Script](#pre-process-script)  
+ - [Post-Process Script](#post-process-script)  
+ - [Rules](#rules)  
+ ---  
+ ## About This Package  
+ 
+**This function checks URLs against [PhishTank](https://www.phishtank.com/) database to see URL is flagged as Phishing or not Phishing.**
+* It can be called on :   
+   * URL Artifacts of Resilient   
+
+  
+After successful completion of checking phishing status on given URL, updates the artifact description field with result status message.
+  
+### PhishTank Function Layout:  
+![screenshot](screenshots/1.png)
+    
+### PhishTank Inputs:  
+![screenshot](screenshots/2.png)  
+  
+### PhishTank Pre-Process Script :
+![screenshot](screenshots/3.png)  
+  
+### PhishTank Post-Process Script :
+![screenshot](screenshots/4.png)
+
+## Prerequisites
+* Resilient Appliance >= v31.0.0  
+* Integrations Server running resilient_circuits >= v30.0.0
+* python-dateutil >= v2.8.0  
+* requests >=  2.21.0
+
+## Installation
+This package requires that it is installed on a RHEL or CentOS platform and uses the resilient-circuits framework.  
+* Download the `.zip` file from our App Exchange and extract it. You will find a file called: `fn_phish_tank-<version>.tar.gz`  
+* Copy this file to your Integrations Server  
+ * To install the package, run:     
+ `pip install fn_phish_tank-<version>.tar.gz`  
+ * To import the function, example rules,data tables and workflows into your Resilient	appliance, run:   
+   `resilient-circuits customize -y -l fn-phish-tank`    
+ * To uninstall IOC Parser Function from Resilient Run the following:    
+    `pip uninstall fn_phish_tank`
+    
+## Resilient Configuration 
+
+Run the following command to generate the PhishTank  configuration section in the app.config file:    
+    
+` resilient-circuits config [-u | -c]`   
+
+The following phish tank configuration data is added:    
```

Comment:
Everywhere you mention "phish tank" can you replace with PhishTank

Reply:
ok